gsc/explore_lc_orbit.py

```diff
@@ -145,10 +145,10 @@ def queued_orbit_search(init_graph, local_ops, save_edges, verbose):
         visited += 1
         # Gets next graph on queue and finds representative nodes
         graph_label = queue.pop()
-        graph = class_graph.node[graph_label]['nx_graph']
+        graph = class_graph.nodes[graph_label]['nx_graph']
         node_equivs = find_rep_nodes(graph)
         # Applies set of local ops to each representative node
-        for rep_node, equiv_nodes in node_equivs.iteritems():
```

Comment:
What was the reasoning for replacing the generator over key-value items with the explicit generation? For speed purposes it's usually better to keep things as generators unless there's a reason for generating the whole set at once.

Reply:
Ah, I see that iteritems is now deprecated, nevermind!